app/views/components/_user-intent-banner.html.erb

```diff
@@ -0,0 +1,4 @@
+<%= tag.div class: "app-c-user-intent-banner" do %>
```

Comment:
There's a few other partials in this folder that are using dashes rather than underscores. Do you think we should leave them all as is or change them all in a subsequent commit?

Reply:
I think that can be a separate PR, but is still good to fix.

Reply:
nice, have changed the name for this one. Will raise another PR after :)